packages/orbit-components/src/Layout/index.tsx

```diff
@@ -18,30 +15,20 @@ const getChildrenProps = (type: Type, key: string) => {
   return null;
 };
 
-const StyledLayout = styled(Grid)`
-  box-sizing: border-box;
-  margin: 0 auto;
-  width: 100%;
-  padding: 0 ${({ theme }) => theme.orbit.spaceMedium};
-
-  ${mq.desktop(css`
-    padding: ${({ theme }) => theme.orbit.spaceLarge};
-  `)};
-`;
-
-StyledLayout.defaultProps = {
-  theme: defaultTheme,
-};
-
 const Layout = ({ children, type, dataTest }: Props) => (
-  <StyledLayout {...LAYOUT_SETTINGS[type]} dataTest={dataTest}>
+  <Grid
+    {...LAYOUT_SETTINGS[type]}
+    width="100%"
+    className="px-md de:p-lg mx-auto my-0 box-border py-0"
```

Comment:
StyledLayout before had "width: 100%", not it does not have this class, could you add? That should fix it

Reply:
tried that, didn't work

Reply:
found the [old build from may](https://9ca42447.orbit.pages.dev/components/layout/layout/) (where it's supposed to be ok), so I guess it's correct, that on tablet and above it does not have full width